src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt

```diff
@@ -64,12 +64,14 @@ class RsInlineFunctionDialog(
     override fun getInlineThisText(): String =
         RefactoringBundle.message("this.invocation.only.and.keep.the.method")
 
-    override fun getKeepTheDeclarationText(): String? =
-        if (function.isWritable && (occurrencesNumber > 1 || !myInvokedOnReference)) {
+    override fun getKeepTheDeclarationText(): String? {
+        val hasMoreThanOneOccurrence = occurrencesNumber < 0 || occurrencesNumber > 1
```

Comment:
I would recommend choosing a different name for this variable such as `mightHaveMultipleOccurences`, and adding a comment about when `occurrencesNumber` is negative